Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -714,6 +714,47 @@ NEVER_INLINE NO_RETURN_DUE_TO_CRASH static void deliberateCrashForTesting()
 #endif
 
 #if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+static void prewarmLogs()
+{
+    static std::array<std::pair<const char*, const char*>, 29> logs { {
```

Comment:
It it feasible at all to keep this list up to date in the future? Seems like we'd be getting performance regressions whenever something else needs to be added, and investigating those is time consuming and unreliable.

Perhaps I'm not aware of some reason to block logging, but perhaps now is the time to reconsider the plan, and to just unblock it? Logs are useful.

Reply:
> It it feasible at all to keep this list up to date in the future? Seems like we'd be getting performance regressions whenever something else needs to be added, and investigating those is time consuming and unreliable.
> 
> Perhaps I'm not aware of some reason to block logging, but perhaps now is the time to reconsider the plan, and to just unblock it? Logs are useful.



> It it feasible at all to keep this list up to date in the future? Seems like we'd be getting performance regressions whenever something else needs to be added, and investigating those is time consuming and unreliable.
> 
> Perhaps I'm not aware of some reason to block logging, but perhaps now is the time to reconsider the plan, and to just unblock it? Logs are useful.

That is a good point! I think we can replace the container manager entries with a call to topPrivatelyControlledDomain(), since that is the call that ends up creating these logs, since the CFNetwork API used in that function contains container manager calls. I will look into that in a follow-up patch. We are blocking the log daemon, but not the WebContent logs themselves. These are now logged via the Networking process.

Thanks for reviewing!